src/NRedisStack/Json/IJsonCommands.cs

```diff
@@ -0,0 +1,232 @@
+﻿using StackExchange.Redis;
+
+namespace NRedisStack;
+
+public interface IJsonCommands
+{
+    /// <summary>
+    /// Appends the provided items to the array at the provided path.
+    /// </summary>
+    /// <param name="key">The key to append to</param>
+    /// <param name="path">The path to append to</param>
+    /// <param name="values">the values to append</param>
+    /// <returns>The new array sizes for the appended paths</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.arrappend"/></remarks>
+    public long?[] ArrayAppend(RedisKey key, string? path, params object[] values);
+    
+    /// <summary>
+    /// Finds the index of the provided item within the provided range
+    /// </summary>
+    /// <param name="key">The key to look up.</param>
+    /// <param name="path">The json path.</param>
+    /// <param name="value">The value to find the index of.</param>
+    /// <param name="start">The starting index within the array. Inclusive.</param>
+    /// <param name="stop">The ending index within the array. Exclusive</param>
+    /// <returns>The index of the value for each array the path resolved to.</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.arrindex"/></remarks>
+    public long?[] ArrayIndex(RedisKey key, string? path, object value, long? start = null, long? stop = null);
+    /// <summary>
+    /// Inserts the provided items at the provided index within a json array.
+    /// </summary>
+    /// <param name="key">The key to insert into.</param>
+    /// <param name="path">The path of the array(s) within the key to insert into.</param>
+    /// <param name="index">The index to insert at.</param>
+    /// <param name="values">The values to insert</param>
+    /// <returns>The new size of each array the item was inserted into.</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.arrinsert"/></remarks>
+    public long?[] ArrayInsert(RedisKey key, string path, long index, params object[] values);
+    
+    /// <summary>
+    /// Gets the length of the arrays resolved by the provided path.
+    /// </summary>
+    /// <param name="key">The key of the json object.</param>
+    /// <param name="path">The path to the array(s)</param>
+    /// <returns>The length of each array resolved by the json path.</returns>
+    /// <remarks><seealso href="https://redis.io/commands/json.arrlen"/></remarks>
+    public long?[] ArrayLength(RedisKey key, string? path = null);
```

Comment:
The command name should be exactly as it is called in the documentation to avoid confusion.
Should ce called "ArrLen"